Branch-SDK/src/io/branch/referral/ServerRequestCreateUrl.java

```diff
@@ -273,4 +273,11 @@ private void updateShareEventToFabric(String url) {
     boolean isPersistable() {
         return false; // No need to retrieve create url request from previous session
     }
+
+    @Override
+    protected boolean prepareExecuteWithoutTracking() {
```

Comment:
Will this ever be false?

Reply:
This is an overriden method from the base class ServerRequest.  ServerRequest default behavior is in fact false, but for a CreateUrl request -- it is always true per the ticket.